apps/web/package.json

```diff
@@ -34,6 +34,7 @@
     "@calcom/embed-snippet": "workspace:*",
     "@calcom/features": "*",
     "@calcom/lib": "*",
+    "@calcom/platform-types": "*",
```

Comment:
Turbo repo already builds dependencies before building web. And since we're importing types in apps/web, we need to set a explicit dependency here.